tooling/cli/src/command/tokens/index.ts

```diff
@@ -69,6 +69,8 @@ export async function generateThemeTypings({
     if (e instanceof Error) {
       console.error(e.message)
     }
+    spinner.stop()
+    process.exit(1)
   } finally {
     spinner.stop()
```

Comment:
Just wanted to note that we have to duplicate `spinner.stop()` calls here because `process.exit` blows away the call stack, so we never reach the `finally` block.